python/src/StandardEvent_doc.i.in

```diff
@@ -43,14 +43,15 @@ correlation matrix. The usual isoprobabilistic transformations are the
 Generalized Nataf transformation and the Rosenblatt one.
```

Comment:
```suggestion
 Generalized Nataf transformation and Rosenblatt transformation.
```
I think the repetition is worth it in this case for the sake of clarity.

Reply:
Fixed.